Sources/PT.PM.Common/MessagePack/RootUstMessagePackSerializer.cs

```diff
@@ -157,18 +157,7 @@ private int SerializeUst(ref byte[] bytes, int offset, Ust value)
             foreach (PropertyInfo property in serializableProperties)
             {
                 object serializeObject = property.GetValue(value);
-                Type propertyType;
-
-                // TODO: backward compatibility with old serialization format (wait for CLangs fix)
-                if (property.Name == nameof(Ust.TextSpan))
-                {
-                    serializeObject = new[] {(TextSpan) serializeObject};
-                    propertyType = typeof(TextSpan[]);
-                }
-                else
-                {
-                    propertyType = property.PropertyType;
-                }
+                Type propertyType = property.PropertyType;
```

Comment:
This var can be removed (paste `property.PropertyType` directly to method argument)